Makefile

```diff
@@ -3,10 +3,10 @@ DOCKER := $(shell which docker)
 DOCKER_COMPOSE := ${DOCKER} compose
```

Comment:
This includes changes from a different PR, they need to be removed before this lands.

Reply:
Yep, that will happen at the end, when everything else is sorted.